deploy/crds/operator.ibm.com_v1alpha1_catalogui_cr.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: operator.ibm.com/v1alpha1
 kind: CatalogUI
 metadata:
   name: catalog-ui
+  annotations:
+    helm.operator-sdk/upgrade-force: "True"
```

Comment:
what's the reason for adding back the annotation?

Reply:
the annotation now works in helm v1.0.0